schemas/common/b2b-lead.schema.json

```diff
@@ -0,0 +1,118 @@
+{
+  "meta:license": [
+    "Copyright 2020 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/common/b2b-lead",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "B2B Lead",
+  "type": "object",
+  "meta:extensible": true,
+  "meta:abstract": true,
+  "description": "This Data Type is used to capture B2B Lead specific attributes.",
+  "definitions": {
+    "b2b-lead": {
```

Comment:
Remove the word "lead" from the property definitions as these properties will always be used under the enclosing scope of xdm: lead. This should be all properties in this data type.